src/components/CreateCoupons/index.jsx

```diff
@@ -0,0 +1,226 @@
+import React, { useState } from "react";
+import "./style.css";
+import { Typography, Button, Row, Col, Avatar, message } from "antd";
+import { InfoCircleOutlined } from "@ant-design/icons";
+import { useTranslation } from "react-i18next";
+import firebase from "../../firebaseConfig";
+
+const db = firebase.firestore();
+const timestamp = firebase.firestore.FieldValue.serverTimestamp;
+const { Title, Text } = Typography;
+
+const CreateCoupons = ({ isTesting }) => {
+  const { t } = useTranslation();
+  const [count, setCount] = useState(1);
+  const [newCoupons, setNewCoupons] = useState({
+    compName: "",
+    amounts: {
+      25: false,
+      50: false,
+      75: false,
+      100: false,
+      150: false,
+      200: false,
+    },
+  });
+  const [ghosted, setGhosted] = useState({
+    "25₺": true,
+    "50₺": true,
+    "75₺": true,
+    "100₺": true,
+    "150₺": true,
+    "200₺": true,
+  });
+  const postCoupons = () => {
+    // db.collection("restaurants").where("uid", "==", "")
+    db.collection("coupons")
+      .add({
+        compName: newCoupons.compName,
+        amounts: { ...newCoupons.amounts },
+        createdAt: timestamp(),
+        img:
+          "https://image.freepik.com/free-photo/wooden-board-empty-table-top-blurred-background_1253-1584.jpg",
```

Comment:
can we import this image and name it ?

Reply:
the link needs to be in the collection. But for other components I can import

Reply:
After we get the auth, we will put a reference there not an img, so it's like a placeholder there for now